pkgs/development/haskell-modules/configuration-nix.nix

```diff
@@ -94,6 +94,8 @@ self: super: builtins.intersectAttrs super {
   # Won't find it's header files without help.
   sfml-audio = appendConfigureFlag super.sfml-audio "--extra-include-dirs=${pkgs.openal}/include/AL";
 
+  cachix = enableSeparateBinOutput super.cachix;
```

Comment:
Think this should be in configuration-nix?

Reply:
I'm not sure, could be also in `common`?